src/peft/tuners/lora/config.py

```diff
@@ -120,6 +120,113 @@ def __post_init__(self):
             raise ValueError("`tau` must be between 0.0 and 1.0.")
 
 
+@dataclass
+class CordaInitConfig:
```

Comment:
I have a suggestion: Can we completely remove the `CordaInitConfig`? It should not be serialized anyway when we save the model, so having a config is not really useful. And the `run_model` and `hooked_model` arguments can be directly passed to `preprocess_corda`, right?